### PR TITLE
multi_disk_random_hotplug:fix no free root port issue

### PR DESCRIPTION
**Pull request by zhenyzha**
It need to apply enough pcie_extra_root_port for the
hotplug operation under arm64.

ID:2117145
Signed-off-by: zhenyzha <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
@PaulYuuu Could you pls help review when you are free. thanks.

```
 (1/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types.arm64-pci: STARTED
 (1/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types.arm64-pci: SKIP: usb controller usb-ehci not available
 (2/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type.arm64-pci: STARTED
 (2/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type.arm64-pci: PASS (929.46 s)
 (3/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types.arm64-pci: STARTED
 (3/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types.arm64-pci: SKIP: usb controller usb-ehci not available
 (4/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type.arm64-pci: STARTED
 (4/4) Host_RHEL.m9.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type.arm64-pci: PASS (943.36 s)
```

File: qemu/tests/cfg/hotplug_unplug_during_io_repeat.cfg [hunk 1]
```diff
@@ -14,7 +14,7 @@
     iozone_timeout = 7200
     sleep_time = 30
     repeat_time = 100
-    q35:
+    q35, arm64-pci, arm64-mmio:
```
Comment (PaulYuuu) [inline qemu/tests/cfg/hotplug_unplug_during_io_repeat.cfg:17]: mmio should not use PCIE bus, I think can move it out.

Comment (zhenyzha) [reply qemu/tests/cfg/hotplug_unplug_during_io_repeat.cfg]: Thanks for the reminder, corrected.

**Comment by zhenyzha** [comment]
@PaulYuuu Could you pls help review again when you are free. thanks.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**